src/main/java/time/TimeInCapitals/showtime/LocalTimeEndpoint.java

```diff
@@ -35,4 +34,41 @@ public String get(@PathVariable(required = false) String timeOrDate) throws Exce
 
 	}
 
+	@GetMapping("/america/{capitalCity}")
```

Comment:
A nie możemy zastosować dwóch różnych `@PathVariable` zamiast tworzyć trzy oddzielne endpointy, które robią tak naprawdę to samo?

Reply:
OK, poprawię.

Reply:
Wstępnie poprawione.